scripts/analytics/addon_snapshot.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from modularodm import Q
+
+from website.settings import ADDONS_AVAILABLE
```

Comment:
When I try to run this script standalone, it fails on this line.  `ADDONS_AVAILABLE` isn't computed until after `init_app()` is run.